### DISCUSSION TITLE
Holy C? Exotic may be the word.

### DISCUSSION THREAD

**Discussion by ed737** [PlummersSoftwareLLC/Primes#528] [Ideas]
Hi Everyone,

I am doing a comp science and Engineering degree at the moment but still have a lot to learn. I saw this race the other day and am doing a unit on different languages at the moment (wrote my first FORTRAN program the other day. That was different, columns matter, like a text file is a matrix haha).

I was wondering if anyone knows of Temple OS and the "Holy C" language. I ask because I know I will not have time to do it (very busy on learning some electrical engineering stuff at the moment) and don't have the OS knowledge to get the homebrew compiler etc working in Linux. I just thought it might be a really interesting addition in the "Exotic languages" section of the project. Some of the syntax and functional changes look interesting. I just thought I'd throw the idea out if anyone with the time wants to give it a go. I thought it might be interesting. 
I apologise if anyone's already bought this up, new to github but I really love the idea behind this project.

Kind regards.

**Comment by marghidanu** [comment]
That would be interesting, but I don't know if you can run TempleOS in Docker, really curious about that.

**Comment by ed737** [comment]
Hi Tudor,

I simplicity dont have the time to do it. The guy had some interesting ideas though. When I was in highschool in the 90s I remember a 5 tiered classification on languages as:

1 Binary
2 Assembly
3 Low level like c
4 Mem cleaning like java
"Enter your natual language here..." (dont need a 5 haha)

I like what he did with taking print out of the equation and if you write a string of something it just prints.
Hes a bit of a cowboy haha.

Also (dangerous) but. Nested switch cases are interesting (especially with threading). I think if compiled in the same arena it would be different. I kinda have a heap of respect for the guy, even though he was a bit of a tool.

Its just so different from anything else. I think its important. It was his life, he wrote some cool shit here. I just wanna see how it performs.

Regards

Ed

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
